peps/pep-0696.rst

```diff
@@ -253,14 +242,14 @@ Using a type parameter from an outer scope as a default is not supported.
 Bound Rules
 ~~~~~~~~~~~
 
-``T2``'s bound must be a subtype of ``T1``'s bound.
+``T1``'s bound must be a subtype of ``T2``'s bound.
 
 ::
 
-   T = TypeVar("T", bound=float)
-   TypeVar("Ok", default=T, bound=int)        # Valid
-   TypeVar("AlsoOk", default=T, bound=float)  # Valid
-   TypeVar("Invalid", default=T, bound=str)   # Invalid: str is not a subtype of float
+   T1 = TypeVar("T1", bound=int)
+   TypeVar("Ok", default=T1, bound=float)        # Valid
+   TypeVar("AlsoOk", default=T1, bound=int)  # Valid
+   TypeVar("Invalid", default=T1, bound=str)   # Invalid: int is not a subtype of str
```

Comment:
There are more indentation issues, though no point worrying about them until the above is resolved (this is a note to self)